website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -44,6 +44,38 @@ Use color logically.
 - **Warning** to help users avoid an issue. Provide guidance and actions, if possible.
 - **Critical** to indicate critical errors that need immediate action.
 
+## Critical Alert vs Critical Toast
```

Comment:
[suggestion] It may make sense to nest this section under "Color" since it discusses a specific color variant.

Reply:
+1